sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubConsumerAsyncClientIntegrationTest.java

```diff
@@ -423,9 +423,10 @@ public void receivesMultiplePartitions() {
 
         // This is the one we'll mutate.
         final Set<Integer> expectedPartitions = new HashSet<>(allPartitions);
+        final Set<Integer> matchPartitions = Collections.synchronizedSet(new HashSet<>(allPartitions));
         final int expectedNumber = 6;
 
-        Assumptions.assumeTrue(expectedPartitions.size() <= expectedNumber,
+        Assumptions.assumeTrue(expectedPartitions.size() == expectedNumber,
```

Comment:
Is there a reason we updated this condition? It doesn't match the error message. (Though, the error message is incorrect and should at least be updated to: "Cannot run this test if there are less partitions than expected.")

Reply:
Fixed in the new version.